README.md

```diff
@@ -0,0 +1,69 @@
+Configuration files for Intel AudioVoiceSpeech (AVS) driver
+========================
+
+ * [Overview](#overview)
+   * [DMIC](#dmic)
+   * [HDAudio](#hdaudio)
+   * [HDMI](#hdmi)
+   * [I2S](#i2s)
+
+# Overview
+
+Each configuration file represents an audio stream topology i.e.: connections of AudioDSP pipelines
+and processing modules that provide rich user experience with the Intel AVS stack. The avs-driver is
+part of the Linux kernel sound subsystem, `sound/soc/intel/avs`. If you do not possess clone of
+kernel's tree, you can view it by visiting
+[Bootlin](https://elixir.bootlin.com/linux/latest/source/sound/soc/intel/avs).
```

Comment:
Instead of bootlin it is probably better to link kernel.org directly?
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/sound/soc/intel/avs

Reply:
Ack.